libs/elements/cards/src/lib/cards.routing.ts

```diff
@@ -0,0 +1,13 @@
+import { NgModule } from '@angular/core'
+import { RouterModule, Routes } from '@angular/router'
+import { ElewaInfoCardComponent } from './components/elewa-info-card/elewa-info-card.component';
+
+export const CARDS_ROUTES: Routes = [
+    { path: '', component: ElewaInfoCardComponent}
+];
+
```

Comment:
No need for routing on this module, we don't have a root path that renders or links to this module

Reply:
I have fixed this one too. Thank you for the heads up